scripts/core/editor3/components/embeds/EmbedBlock.tsx

```diff
@@ -5,6 +5,10 @@ import {connect} from 'react-redux';
 import {QumuWidget, isQumuWidget} from './QumuWidget';
 import * as actions from '../../actions';
 import ng from 'core/services/ng';
+import {loadIframelyEmbedJs} from './loadIframely';
+import {debounce} from 'lodash';
+
+const loadIframely = debounce(loadIframelyEmbedJs, 100);
```

Comment:
You forgot to remove debouncing. Is there a reason why do you rebase commits? I don't like it mostly for having to review the entire thing again. There is not much to review in this PR, but I also don't get notifications from github about new commits when you push it.

Reply:
this should still be there, otherwise on first load this would call widgets.load more than once

Reply:
I think I now understand why you use debouncing here. When there is only one embed in the editor3, it's only called once. When there are multiple embeds it's called once per embed which seems is what you are trying to prevent. Is that right?

Why can't you call `widgets.load` more than once on the first load? Is anything breaking due to it or you are just saving http requests?